src/java/org/apache/cassandra/service/accord/AccordKeyspace.java

```diff
@@ -537,6 +581,20 @@ public static Mutation getCommandMutation(AccordCommandStore commandStore, Accor
         }
     }
 
+    public static ByteBuffer serializeToken(Token token)
+    {
+        return serializeToken(token, ByteBufferAccessor.instance);
+    }
+
+    private static <V> V serializeToken(Token token, ValueAccessor<V> accessor)
```

Comment:
I could have written this off `ByteBuffer`, but had to do a lot of `ValueAccessor` work recently so was more habit than anything